.github/workflows/deploy.yml

```diff
@@ -4,63 +4,62 @@ name: Release Artifacts
 on:
   push:
     tags:
-      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v*" # Push events to matching v*, i.e. v1.0, v20.15.10
 jobs:
   build-jar:
     name: Build Effekt compiler and run tests
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "true"
 
-    - uses: actions/checkout@v2
-      with:
-        submodules: 'true'
-
-    - name: Set up JDK 11
-      uses: actions/setup-java@v1
-      with:
-        java-version: 11
-
-    - name: Set up NodeJS
-      uses: actions/setup-node@v1
-      with:
-        node-version: '12.x'
-
-    - name: Install MLton
-      run: |
-        curl -L https://github.com/MLton/mlton/releases/download/on-20210117-release/mlton-20210117-1.amd64-linux-glibc2.31.tgz --output mlton.tgz
-        tar -xzf mlton.tgz
-        mv mlton-20210117-1.amd64-linux-glibc2.31 $GITHUB_WORKSPACE/mlton
-        chmod +x $GITHUB_WORKSPACE/mlton/bin/mlton
-        echo "Trying to call directly"
-        $GITHUB_WORKSPACE/mlton/bin/mlton
-        echo "Adding mlton to path"
-        echo "$GITHUB_WORKSPACE/mlton/bin" >> $GITHUB_PATH
-
-    - name: Install Chez Scheme
-      run: sudo apt-get install chezscheme
-
-    - name: Run tests and assemble jar file
-      run: sbt clean deploy
-
-    - id: npmpackage
-      name: Generate npm package
-      run: mv $(npm pack) effekt.tgz
-
-    - name: Upload the npm package
-      uses: actions/upload-artifact@v1
-      with:
-        name: effekt-npm-package
-        path: effekt.tgz
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Set up NodeJS
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: Install MLton
+        run: |
+          curl -L https://github.com/MLton/mlton/releases/download/on-20210117-release/mlton-20210117-1.amd64-linux-glibc2.31.tgz --output mlton.tgz
+          tar -xzf mlton.tgz
+          mv mlton-20210117-1.amd64-linux-glibc2.31 $GITHUB_WORKSPACE/mlton
+          chmod +x $GITHUB_WORKSPACE/mlton/bin/mlton
+          echo "Trying to call directly"
+          $GITHUB_WORKSPACE/mlton/bin/mlton
+          echo "Adding mlton to path"
+          echo "$GITHUB_WORKSPACE/mlton/bin" >> $GITHUB_PATH
+
+      - name: Install Chez Scheme
+        run: sudo apt-get install chezscheme
+
+      - name: Run tests and assemble jar file
+        run: sbt clean deploy
```

Comment:
@jiribenes just told me MacOS minutes are much more expensive. Maybe we don't want to run the tests on all systems??? :)